sections/telemetry_system.tex

```diff
@@ -395,12 +442,36 @@ \subsection{Data Transmission}
 
 In order to transmit telemetry data over radio, \gls{cuinspace} has developed a radio packet encoding format. This
 format can be found in Appendix \ref{apx:comm-spec-repos}. The format covers multiple different data types that are
-recorded about the rocket state, such as altitude, acceleration, \gls{gps} coordinates, etc. \cite{radio-comms}.
+recorded about the rocket state like altitude, acceleration, or \gls{gps} coordinates. \cite{radio-comms}
 Importantly, each radio packet starts with a header containing the amateur radio call-sign of the \gls{cuinspace}
 operator responsible for flying the system. \cite{radio-comms} The original specification did not include sufficient
 space to append a call-zone indicator to the call-sign, which has since been amended in the latest version. This is a
 requirement for Canadians broadcasting in the United States \cite{foreign-broadcast}, as \gls{cuinspace} does at
-\gls{sac}.
+\gls{sac}. \gls{cuinspace} plans on using this same packet encoding in its data logging for consistency and compatability
```

Comment:
Nitpick but you could mention that for logging we omit the call-sign to save more space!